integration-tests/liquibase/src/main/resources/db/changeLog.xml

```diff
@@ -2,7 +2,7 @@
         xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-                        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+                        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.14.xsd">
```

Comment:
Replace this with `latest` to avoid maintenance burden?

Reply:
See https://github.com/quarkusio/quarkus/pull/26977#discussion_r931653897

To quote from https://github.com/liquibase/liquibase/releases/tag/v4.12.0:
> The uploaded "latest" XSD will always be the most recent Liquibase release, so note how they can be different and locally cached IDE versions may be different yet.

So things can deviate. OTOH, I don't think that these files are touched that often in IDEs (or even at all).
Liquibase will _not_ download newer xsds by default, see https://github.com/liquibase/liquibase/pull/2558. So that part is covered.

Long story short, I'll switch to `latest` and we'll see how it goes.